app/src/main/java/co/nimblehq/alpine/sample/CameraCaptureActivity.kt

```diff
@@ -0,0 +1,98 @@
+package co.nimblehq.alpine.sample
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import androidx.activity.ComponentActivity
+import androidx.camera.core.*
+import androidx.camera.lifecycle.ProcessCameraProvider
+import androidx.core.content.ContextCompat
+import co.nimblehq.alpine.databinding.ActivityCameraCaptureBinding
+import co.nimblehq.alpine.sample.extension.getStatusBarHeight
+import co.nimblehq.alpine.sample.extension.setLightStatusBar
+
+open class CameraCaptureActivity : ComponentActivity() {
+
+    companion object {
+
+        fun start(activity: Activity) {
+            val intent = Intent(activity, CameraCaptureActivity::class.java)
+            activity.startActivity(intent)
+        }
+    }
+
+    private lateinit var binding: ActivityCameraCaptureBinding
+    private val bindingInflater: (LayoutInflater) -> ActivityCameraCaptureBinding
+        get() = { inflater -> ActivityCameraCaptureBinding.inflate(inflater) }
+
+    private var camera: Camera? = null
+    private var imageCapture: ImageCapture? = null
+    private var preview: Preview? = null
+    private var cameraProvider: ProcessCameraProvider? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setLightStatusBar()
+        setContentView(bindingInflater.invoke(layoutInflater).apply {
```

Comment:
Why don't we directly call `ActivityCameraCaptureBinding.inflate()` here and get rid of the class val? 🤔 
```suggestion
        setContentView(ActivityCameraCaptureBinding.inflate(layoutInflater).apply {
```

Reply:
good point @ryan-conway 👍